src/main.rs

```diff
@@ -20,7 +20,12 @@ use teleport::{
 #[structopt(name = "teleport", about = "A tool for CoinSwap")]
 struct ArgsWithWalletFile {
     /// Wallet file
-    #[structopt(default_value = "wallet.teleport", parse(from_os_str), long)]
+    #[structopt(
+        default_value = "wallet.teleport",
+        parse(from_os_str),
+        long,
+        short = "w"
+    )]
```

Comment:
There's an issue https://github.com/utxo-teleport/teleport-transactions/issues/5 to convert from structopt to clap 3.0. With improves short naming and helper docs. 

Would you like to tackle that instead of just this single change?

Reply:
Yeah, I'll take it on. I'll close this PR and have them in separate solutions.